service_mesh/v2x/servicemesh-release-notes.adoc

```diff
@@ -13,6 +13,8 @@ Red Hat is committed to replacing problematic language in our code, documentatio
 
 include::modules/ossm-rn-new-features.adoc[leveloffset=+1]
 
+include::modules/ossm-config-disable-networkpolicy.adoc[leveloffset=+2]
```

Comment:
I'm of two minds here, maybe we want to change the leveloffset to +1 to bump this out for visibility since we can't put it right under the 2.1.1 heading.